usr/share/rear/prep/NFS4SERVER/default/400_prep_nfs_server.sh

```diff
@@ -0,0 +1,13 @@
+# 400_prep_nfs_server.sh
+#
+# prepare stuff for NFS4SERVER
+#
+
+PROGS+=("${PROGS_NFS4SERVER[@]}")
+
+REQUIRED_PROGS+=("${REQUIRED_PROGS_NFS4SERVER[@]}")
+
+MODULES_LOAD+=("${NFS4SERVER_MODULES[@]}")
+
+# Check if at least one trusted client was specified
+(( "${#NFS4SERVER_TRUSTED_CLIENTS[@]}" > 0 )) || Error "You must have defined at least one client in NFS4SERVER_TRUSTED_CLIENTS."
```

Comment:
Only to be on the safe side:
I assume it is OK that NFS4SERVER_TRUSTED_CLIENTS could be empty
during "rear recover" (e.g. the user can edit etc/rear/local.conf
inside the ReaR recovery system before he runs "rear recover").

Scripts in the 'prep' stage are run during "rear mkrescue/mkbackup"
but they are not run during "rear recover".
During "rear recover" scripts in the 'verify' stage are run.
```
# rear -s mkrescue
...
Source conf/...
Source init/...
Source prep/...
Source layout/...
Source rescue/...
Source build/...
Source pack/...
Source output/...

# rear -s recover
...
Source conf/...
Source init/...
Source setup/...
Source verify/...
Source layout/...
Source restore/...
Source finalize/...
Source wrapup/...
```

Reply:
Yes, good point. Need to check during `prep` and `verify` stages.